frontend/src/Frontend/ReplGhcjs.hs

```diff
@@ -348,3 +351,23 @@ headerBtnCfg
   :: (Default (UiButtonCfgRep f), IsString (ReflexValue f CssClass), Semigroup (ReflexValue f CssClass))
   => UiButtonCfgRep f
 headerBtnCfg = btnCfgPrimary & uiButtonCfg_class %~ (<> "main-header__button")
+
+resourcesWidget
+  :: (DomBuilder t m)
+  => m ()
+resourcesWidget = elClass "div" "resources" $ do
+  resourceCell "Support" (static @"img/resources/support.svg") "#"
+    "Explore Help Resources to learn about Chainweaver, solve problems and get in touch"
+  resourceCell "Documentation" (static @"img/resources/documentation.svg") "http://pact-language.readthedocs.io/"
+    "Complete technical resources for Chainweaver, Kadena blockchain and Pact language"
+  resourceCell "Tutorials" (static @"img/resources/tutorials.svg") "https://pactlang.org/"
+    "Read or watch tutorials for writing smart contracts using the Pact language"
+  where
+    resourceCell title iconUrl href desc =
+      elAttr "a" ("class" =: "resources__cell" <> "href" =: href <> "target" =: "_blank")  $ do
+        elAttr "div" ("class" =: "resources__cell-icon" <> "style" =: ("background-image: url(" <> iconUrl <>")")) $ blank
```

Comment:
Do we ever use the impure static obelisk urls (i.e the ones without the hash in the uri that redirect) as links in our sass files? It's a bit weird putting this out in an inline style just because we need to be in haskell to use the static function.

Reply:
Oh yeah, we totally do in fonts. Is that preferred over the inline style here?

Reply:
> It's a bit weird putting this out in an inline style just because we need to be in haskell to use the static function.

AFAICT there are two ways to do this in haskell without inline style. 
You can use classes here and then either throw the CSS in a `<style>` in [<head>](https://github.com/kadena-io/chainweaver/pull/197/commits/5e0475dcfb2ab90adc9656ba149f69ad8e64ccec#diff-9f2ed08cd073fc795038ebf78b9b0137R104) tag or use some EDSL CSS preprocessor. 
Personally I'm not opposed to inline style but for consistency I'd recommend the `<head>` route.